addon-test-support/-private/mock-window.js

```diff
@@ -15,5 +15,5 @@ export function mockWindow(scope) {
     throw new Error('mockWindow must be called from an unit/integration test!');
   }
   
-  return register('service:window', windowMockFactory(), { instantiate: false });
+  return register.call(this, 'service:window', windowMockFactory(), { instantiate: false });
```

Comment:
You are right, should be `scope` insteaf of `this` I think!

Reply:
Fingers crossed!